build/ProjectReunion-BuildFoundation.yml

```diff
@@ -232,7 +232,20 @@ jobs:
 #      dir /s $(Build.ArtifactStagingDirectory)\symbols
 
   - task: PublishSymbols@2
-    displayName: 'Publish symbols (without source indexing)'
+    displayName: 'Publish private symbols to internal server (without source indexing)'
+    inputs:
+      searchPattern: '$(Build.ArtifactStagingDirectory)\symbols\**\*.pdb'
+      symbolServerType: 'TeamServices'
+# this ADO task does not support indexing of github sources currently :-(      
+      indexSources: false 
+      detailedLog: true
+    # There is a bug which causes this task to fail if LIB includes an inaccessible path (even though it does not depend on it).
+    # To work around this issue, we just force LIB to be any dir that we know exists.
```

Comment:
Nit: Is there a bug number tracking this? I'd add it if so. That would give folks a method of quickly checking up on the bug to answer the question "can I delete these lines now?"

Reply:
Not that I'm aware of. This comment is actually just copy pasted from the lines above, and I'm not the original author of the other one. 🤷‍♂️